### PR TITLE
Design user dashboard

### PR DESCRIPTION
**Pull request by Admayah**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -2,42 +2,17 @@
...
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
-    <title>React App</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,700;0,900;1,500&display=swap" rel="stylesheet">
+    <title>PropertyPro App</title>
   </head>
   <body>
```
Comment (mykelmatics) [inline public/index.html:14]: Why did you remove the noscript tag?

File: src/components/Herosection/hero.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  background-repeat: no-repeat;
+  background-size: cover;
+  object-fit: cover;
+}
+
+.hero-tagline-container {
+  width: 50%;
+  padding: 2rem 2.1875rem 4.125rem;
+  width: 80%;
+  color: white;
```
Comment (mykelmatics) [inline src/components/Herosection/hero.css:20]: why using two width ?

File: src/components/Herosection/hero.css [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+.hero-tagline-header {
+  margin-bottom: 10px;
+}
+.hero-sub-tagline {
+  margin-bottom: 57px;
+  line-height: 1.3;
+}
+.tagline-text {
+  font-size: 50px;
+  margin-bottom: 1rem;
```
Comment (mykelmatics) [inline src/components/Herosection/hero.css:43]: I suggest you use rem instead of px for your font-size

File: src/components/Navbar/navbar.css [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+  color: white;
+  padding-left: 10px;
+  display: flex;
+  align-items: center;
+  z-index: 1000;
+}
+
+.logo {
+  font-size: 30px;
+  color: white;
```
Comment (mykelmatics) [inline src/components/Navbar/navbar.css:23]: I suggest you use rem for the font

File: src/components/Navbar/navbar.css [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+  padding-right: 10px;
+  border-radius: 50%;
+  object-fit: cover;
+}
+.logo-name {
+  font-size: 18px;
```
Comment (mykelmatics) [inline src/components/Navbar/navbar.css:29]: same as here

File: src/components/Navbar/navbar.css [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+
+@media (max-width: 768px) {
+  .hide-navbar-menu {
+    top: 0;
+    background-color: #524141;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    list-style: none;
```
Comment (mykelmatics) [inline src/components/Navbar/navbar.css:68]: I suggest you add the list style to the global styling

File: src/components/Navbar/navbar.css [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+    width: 100%;
+    height: 100vh;
+    z-index: 100;
+    transition: all 0.5s linear;
+  }
+}
+
+.navbar-menu {
+  padding-right: 20px;
+  font-size: 20px;
```
Comment (mykelmatics) [inline src/components/Navbar/navbar.css:80]: Change to rem

File: src/components/OurServicesSection/ourservices.css [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  flex-wrap: wrap;
+}
+@media (max-width: 768px) {
+  .our-services-wrapper {
+    display: flex;
+    flex-direction: column;
+  }
```
Comment (mykelmatics) [inline src/components/OurServicesSection/ourservices.css:20]: I think you are repeating yourself here for the display and flex-direction

File: src/components/OurServicesSection/ourservices.css [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+    border-radius: 0;
+    max-width: 400px;
+  }
+}
+.service-icon {
+  text-align: center;
+}
+.our-service-icon {
+  font-size: 30px;
+}
```
Comment (mykelmatics) [inline src/components/OurServicesSection/ourservices.css:69]: Change to "rem"

File: src/components/allproperties/propertiesdata.css [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+  color: black;
+  margin-top: 20px;
+}
+.agent-no {
+  display: flex;
+  align-items: center;
+}
+.wb-color {
+  margin-right: 5px;
+  font-size: 20px;
```
Comment (mykelmatics) [inline src/components/allproperties/propertiesdata.css:87]: change to "rem"

File: src/components/feed/Feed.jsx [hunk 11]
```diff
@@ -0,0 +1,70 @@
...
+                            <i className="fa fa-bed property-feature"></i>
+                          </span>
+                          {item.features.no_of_garage}
+                        </a>
+                      </li>
+                    </ul>
+                    <div className="property-price">
+                      <span>{item.price}</span>
+                      <button className="k">EDIT</button>
+                      <button className="k">DELETE</button>
```
Comment (mykelmatics) [inline src/components/feed/Feed.jsx:57]: The naming is not descriptive enough

File: src/components/footer/footer.css [hunk 12]
```diff
@@ -0,0 +1,254 @@
...
+.accordion {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  padding-bottom: 20px;
+}
+@media (min-width: 768px) {
+  .accordion {
+    width: 70%;
+    display: flex;
```
Comment (mykelmatics) [inline src/components/footer/footer.css:57]: I suggest you remove the display it has been the one before this screen size should cover for it

File: src/components/footer/footer.css [hunk 13]
```diff
@@ -0,0 +1,254 @@
...
+  display: block;
+  width: 0;
+  height: 2px;
+  transition: width 0.3s;
+  background: #000;
+}
+.property-area:hover::after {
+  width: 100%;
+}
+.li-link {
```
Comment (mykelmatics) [inline src/components/footer/footer.css:123]: I suggest you use a more descriptive name for this

File: src/components/footer/footer.css [hunk 14]
```diff
@@ -0,0 +1,254 @@
...
+  justify-content: space-between;
+  color: white;
+  border-bottom: 1px solid white;
+  opacity: 0.5;
+}
+@media (min-width: 768px) {
+  .social-menu {
+    width: 25%;
+    display: flex;
+    justify-content: space-between;
```
Comment (mykelmatics) [inline src/components/footer/footer.css:140]: I think the justify content you specify earlier should still work for this

File: src/components/footer/footer.css [hunk 15]
```diff
@@ -0,0 +1,254 @@
...
+  flex-wrap: wrap;
+  padding: 0;
+  margin: 0;
+}
+@media (min-width: 768px) {
+  .social-icons {
+    width: 200px;
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
```
Comment (mykelmatics) [inline src/components/footer/footer.css:159]: I think the flex-wrap is not needed here again

File: src/components/footer/footer.css [hunk 16]
```diff
@@ -0,0 +1,254 @@
...
+  display: flex;
+  flex-direction: column-reverse;
+  justify-content: space-between;
+}
+@media screen and (min-width: 768px) {
+  .contact-info-container {
+    width: 200px;
+    padding-bottom: 25px;
+    display: flex;
+    justify-content: space-between;
```
Comment (mykelmatics) [inline src/components/footer/footer.css:187]: you can remove  both the display and justify content here. You have specify it before this  screen

File: src/components/footer/footer.css [hunk 17]
```diff
@@ -0,0 +1,254 @@
...
+  align-items: center;
+  padding: 10px 0;
+}
+@media screen and (min-width: 768px) {
+  .contact-info {
+    padding: 5px 0;
+  }
+}
+.social-contact {
+  font-size: 30px;
```
Comment (mykelmatics) [inline src/components/footer/footer.css:203]: I suggest you use "rem"

File: src/components/footer/footer.css [hunk 18]
```diff
@@ -0,0 +1,254 @@
...
+.copyright-container {
+  color: white;
+  padding: 0 35px;
+  display: flex;
+  justify-content: space-between;
+}
+@media (min-width: 768px) {
+  .copyright-container {
+    display: flex;
+    justify-content: space-between;
```
Comment (mykelmatics) [inline src/components/footer/footer.css:215]: I don't think this needed

File: src/components/footer/footer.css [hunk 19]
```diff
@@ -0,0 +1,254 @@
...
+  display: flex;
+  flex-direction: column-reverse;
+  justify-content: space-between;
+}
+@media (min-width: 768px) {
+  .copyright-text {
+    width: 90%;
+    padding: 0 48px;
+    display: flex;
+    justify-content: space-between;
```
Comment (mykelmatics) [inline src/components/footer/footer.css:243]: Try remove the display and flex-direction

File: src/components/sidebar/sidebar.css [hunk 20]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+.sidebar-menu {
+  display: flex;
+}
+.sidebar-menu:hover{
+  font-size: 50px;
+}
+.sidebar-link{
+  text-decoration: none;
```
Comment (mykelmatics) [inline src/components/sidebar/sidebar.css:56]: I suggest you include this in your global styling

File: src/pages/dashboard/dashboardnav/dashboardnav.css [hunk 21]
```diff
@@ -0,0 +1,89 @@
...
+.dashboard-nav-wrapper {
+  display: flex;
+  justify-content: space-between;
+  padding: 20px;
+}
+@media (max-width: 768px) {
+  .dashboard-nav-wrapper {
+    position: relative;
+    display: flex;
+    justify-content: space-between;
```
Comment (mykelmatics) [inline src/pages/dashboard/dashboardnav/dashboardnav.css:15]: same here

**Review by mykelmatics — COMMENTED** [review]
Overall the code looks good. but dropped a few comments

File: src/components/Herosection/Hero.jsx [hunk 22]
```diff
@@ -0,0 +1,31 @@
...
+      <div className="hero-wrapper">
+        <div className="hero-background-img-container">
+          <div className="hero-tagline-container">
+            <div className="hero-tagline-header">
+              <span className="tagline-text">
+                Own, Live and Love your dream home
+              </span>
+            </div>
+            <p className="hero-sub-tagline">
+              Lorem ipsum dolor, sit amet consectetur adipisicing elit.
```
Comment (nosisky) [inline src/components/Herosection/Hero.jsx:17]: Please put some actual text and not some dummy text.
